docs/pages/enterprise/sso.mdx

```diff
@@ -237,5 +239,44 @@ must be able to:
 If something is not working, we recommend to:
 
 - Double-check the host names, tokens and TCP ports in a connector definition.
-- Look into Teleport's audit log for claim mapping problems. It is usually stored on the
-  auth server in the `/var/lib/teleport/log` directory.ad
+- Review the below log information and the troubleshooting sections for the above guides.
+
+
+### Using the Web UI
+
+If you get "access denied" or other login errors, the number one place to check is the Audit
+Log. You can access it in the **Activity** tab of the Teleport Web UI.
+
+![Audit Log Entry for SSO Login error](../../img/sso/teleportauditlogssofailed.png)
+
+Example of a user being denied because the role `clusteradmin` wasn't set up:
+
+```json
+{
+  "code": "T1001W",
+  "error": "role clusteradmin is not found",
+  "event": "user.login",
+  "method": "oidc",
+  "success": false,
+  "time": "2019-06-15T19:38:07Z",
+  "uid": "cd9e45d0-b68c-43c3-87cf-73c4e0ec37e9"
+}
+```
+
+### No Servers show
+
+Servers will show if a user's roles include a matching server label that includes logins.
```

Comment:
I think this description should be more detailed. Right now, it looks like a user's roles must match a server label that contains a user's login. I also think this description should explain what "show" means, and should connect this more explicitly to SSO, since this section is in the context of SSO troubleshooting.

One suggestion:

```markdown
When Teleport's Auth Service receives a request to list Teleport Nodes,
it only returns the Nodes that a user is authorized to access. 

A user's Teleport roles must grant the user explicit access to Nodes with a particular label
 before the user can view those Nodes. The Auth Service compares the user's 
`traits.logins` with the `allow` and `deny` rules defined in each of the user's roles. 
If the user's logins match a role's `allow` and `deny` rules, and a Node's labels match 
the keys and values listed within a role's `node_labels` field, then the Auth Service will
list the Node in response to the user's request.

When configuring SSO, ensure that the identity provider is populating each user's 
traits correctly. For a user to see a Node in Teleport, the result of populating a
 template variable in a role's `allow.logins` must match at least one of a user's
 `traits.logins`.
```

Reply:
updated